tools/test_backups2datalad/test_zarr.py

```diff
@@ -221,7 +221,7 @@ async def test_backup_zarr_pathological(
         f"{new_dandiset.dandiset.version_api_path}assets/",
         json={"metadata": {"path": "empty.zarr"}, "zarr_id": empty_zarr_id},
     )
-    await arequest(client.session, "POST", f"/zarr/{empty_zarr_id}/ingest/")
+    await arequest(client.session, "POST", f"/zarr/{empty_zarr_id}/finalize/")
```

Comment:
Is old endpoint also working that tests were passing before too?

Reply:
The old endpoint was removed when https://github.com/dandi/dandi-archive/pull/1448 was merged, after which the tests here started failing (though this specific cause was lost in the noise from https://github.com/dandi/dandi-archive/issues/1494).